src/hyper.js

```diff
@@ -26,62 +16,23 @@ export class Hyper extends XdrPrimitiveType {
     return Number(this._value >> 32n) >> 0;
   }
 
-  get unsigned() {
-    return false;
-  }
-
-  toString() {
-    return this._value.toString();
-  }
-
-  toJSON() {
-    return {_value: this._value.toString()}
-  }
-
-  /**
-   * @inheritDoc
-   */
-  static read(reader) {
-    return new Hyper(reader.readBigInt64BE());
+  get size() {
+    return 64;
   }
 
-  /**
-   * @inheritDoc
-   */
-  static write(value, writer) {
-    if (!(value instanceof this))
-      throw new XdrWriterError(`${value} is not a Hyper`);
-    writer.writeBigInt64BE(value._value);
-  }
-
-  /**
-   * Create Hyper instance from string
-   * @param {String} string - Numeric representation
-   * @return {Hyper}
-   */
-  static fromString(string) {
-    if (!/^-?\d{0,19}$/.test(string))
-      throw new TypeError(`Invalid i64 string value: ${string}`);
-    return new Hyper(BigInt(string));
+  get unsigned() {
+    return false;
```

Comment:
Would these be easier to understand if it was
```js
get signed() {
  true
}
```

Instead of phrasing it with the double-negative, of not-unsigned.

Reply:
I guess I saw such approach in some other serialization library. For example, `Long` in `bson` serializer.